editor/src/components/inspector/inspector.tsx

```diff
@@ -348,29 +360,37 @@ export const Inspector = React.memo<InspectorProps>((props: InspectorProps) => {
             display: shouldShowInspector ? undefined : 'none',
           }}
         >
-          <AlignmentButtons numberOfTargets={selectedViews.length} />
+          {unless(
+            isConditionalSelected,
+            <AlignmentButtons numberOfTargets={selectedViews.length} />,
+          )}
           {when(isTwindEnabled(), <ClassNameSubsection />)}
           {anyComponents ? <ComponentSection isScene={false} /> : null}
           <ConditionalSection paths={selectedViews} />
```

Comment:
Hmmm should this only be rendered when a conditional is selected? This feels like the fork in the Inspector logic should be higher up unless there's a reason for it to live here.

This also raises questions about multiselection - e.g. right now if I multiselect a conditional and a DOM element we end up with a very broken looking Inspector

Reply:
> should this only be rendered when a conditional is selected
as far as I know, the Inspector function is basically the outermost place where this logic can be placed.

Regarding the broken looking conditional control, thanks for spotting it! I refactored the code so that all inspector sections are shown when both a DOM element and a conditional is selected (since the conditional control already handles multiselection, and the gist of the PR is preserved when only a single conditional is selected)